Source/WebGPU/WebGPU/Device.mm

```diff
@@ -139,7 +139,7 @@ static void captureFrame(id<MTLDevice> captureObject)
 
 Ref<Device> Device::create(id<MTLDevice> device, String&& deviceLabel, HardwareCapabilities&& capabilities, Adapter& adapter)
 {
-    id<MTLCommandQueue> commandQueue = [device newCommandQueueWithMaxCommandBufferCount:2048];
+    id<MTLCommandQueue> commandQueue = [device newCommandQueueWithMaxCommandBufferCount:4096];
```

Comment:
This number is increased so we don't hang the GPU process while we wait for Javascript's GC to run